lib/puppet_webhook/r10k_log_proxy.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'log4r/outputter/outputter'
+
+module PuppetWebhook
+  # This class creates a proxy that will grab r10k logging and output it to the logger passed into this class
+  class R10KLogProxy < Log4r::Outputter
+    def initialize(logger, level)
+      super('puppet_webhook')
+
+      @logger = logger
+      @logger.level = level
```

Comment:
what's the point of passing logger and level? above we extract level from logger, and here we initialise logger's level that we got passed
are those ever different?

Reply:
Probably not, I was just being safe.